### ISSUE TITLE
How much precise is precision?

### ISSUE THREAD

**Issue by louy** [sunng87/node-geohash#26]
In mathematical terms, how precise is the option `precision`? Like how many meters is precision **9** or **8**? Does the option stand for the number of digits after the decimal point?

**Comment by sunng87** [comment]
It has nothing to do with number, it mean how many characters in the hash string.

**Comment by louy** [comment]
But I believe the number of digits is related to the string length. How many digits does a character fit?
Is it `digitsPerCoordinate ~= hashLength / 2 * 32 / 10` maybe (base32 to decimal)?

**Comment by sunng87** [comment]
If you look into the code, you can find a few lines of code that compute the number of chars from digits after the decimal point. 

https://github.com/sunng87/node-geohash/blob/master/main.js#L65

**Comment by louy** [comment]
I see how it works now.
So what I wanted was to group coordinates that are close to each other together. Apparently this can't be done here with precise control on the error margin.

However, I've created a test script to see how accurate it is. Apparently `10` is precise up to the fifth decimal place or 1.1 meter (error less than 0.00001) and `9` is precise up to the fourth decimal place or 11 meters (error less than 0.0001). `8` would be 110 meters.

One thing I noticed is that the error margin in latitude is higher most of the time, like 70-80% of the time.

And one last note, this doesn't mean that nearby points will have the same hash. So it cannot be used to group things together. If anyone wants to do that he'll have to round the latitude/longitude and then generate a hash.
